README.md

```diff
@@ -0,0 +1,46 @@
+<!--
+Copyright (C) 2022 Nitrokey GmbH
+SPDX-License-Identifier: CC0-1.0
+-->
+
+PIV-Authenticator
+=================
+
+`piv-authenticator` is a Rust implementation of the [Personal Identity Verification](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-73-4.pdf) smartcard.
+
+Supported features
+------------------
+
+Nearly all functionality specified by the standard are implemented.
+
+Non-standard management operations are partially implemented. Basic functionality such as factory reset and configuration of the management key is implemented.
+
+See the [tracking issue for command support](https://github.com/Nitrokey/piv-authenticator/issues/1) for more information.
+
+License
+-------
+
+This project is licensed under the [GNU Lesser General Public License (LGPL)
```

Comment:
We agreed on Apache2 OR MIT.

Reply:
Right, this is a mistake. All files have an Apache2 + MIT header. I will fix this. Are you fine with leaving examples/configs in CC0 ? They're probably not copyrightable anyway and it doesn't make much sense to require attribution when someone copies them in their project. As long as they respect the copyright for the library itself.